website/content/maintainers/templates/governance-elections.md

```diff
@@ -8,6 +8,260 @@ aliases:
+based on who is currently making project decisions. This group will be
+responsible for running the project, but with a set deadline for how they will
+be replaced by an elected steering committee.
+
+**Question: do we have a good example of this that I can link to? I'm failing to
+find the original one from k8s**
+
+### Decision Process
+
+We recommend using consensus as the primary decision-making method, but you will
+also need to determine the list of things that requires a vote, such as issuing
+or amending policies, spending, etc. This section will also specify how you
+determine quorum for voting with recommended language found in the template.
+
+### Getting in touch
+
+Your governance document should specify how community members can raise issues to
+the steering committee for decision. Common ways to do this are via an email
+alias for the Steering Committee, filing an issue in a specific repository, or
+posting in a Steering Committee Slack channel.
+
+### Composition
+
+This section requires some extra work and addition thought put into how you plan
+to handle election terms and replacing Steering Committee members whose terms
+are up for election.
+
+We recommend using staggered 2-year terms where only half of the seats are up
+for election in any given year to maintain continuity. This keeps enough people
+on the Steering Committee to provide context and onboarding for new members.
+
+If you are starting from scratch with a bootstrapped committee, and need a
+transition period, but plan to have staggered 2-year terms, we recommend that for
+the first year of the Steering Committee half of the seats should be elected for a
+1-year term, and the other half should be elected for a 2-year term.
+
+While not recommended you can elect-everyone-at-once and have all of the members
+elected at the same time to serve for a specified term.
+
+### Election Procedure
+
+Note: CNCF will soon have a better, GitOps-driven, online election tool
+available.  At that time, projects will want to revise portions of this
+procedure.
+
+The template contains recommended text for timelines, election officers, and
+eligibiity to vote. These sections are all relatively straightforward, and we
+recommend using the templated text, but you will need to determine the threshold
+for voter eligibility and specify where community members can find the list of
+eligible voters.
+
+**Candidate Eligibility**
+
+You will need to determine what criteria a person needs to meet in order to be
+eligible to run as a candidate in the election. Some common criteria include:
+being in a project member role in the [Contributor Ladder], listed as an
+approver in one or more OWNERS files, or similar.
+
+**Voting Procedure**
+
+For most elections, we recommend using 
+[Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on
+[CIVS](http://civs.cs.cornell.edu/) using the IRV method as specified in the
```

Comment:
I don't agree that we should recommend CIVS.  Elekto, OpaVote, Helios, sure.  But CIVS has enough issues that we shouldn't steer anyone towards it.

Reply:
Hmmm ... do I need to update the template?

Reply:
I'll update this PR with those recommendations, but yes, one of us will need to update the template to match.